src/Workspaces/Core/Portable/Workspace/Solution/Solution.cs

```diff
@@ -1337,5 +1505,68 @@ internal Solution WithOptions(SerializableOptionSet options)
 
             return new Solution(newState);
         }
+
+        private void CheckContainsDocument(DocumentId? documentId)
```

Comment:
Mark as this it asserts documentId is non-null after the call?

Reply:
Will do.